modules/developers-guide/pages/work-with-languages.adoc

```diff
@@ -173,17 +173,17 @@ To modify the template source code:
 As you write your program, keep in mind that there are two types of calls—update calls and query calls—and that update functions use asynchronous messaging.
 . Save your changes and close the `+main.rs+` file.
 
-=== Connect to a network and deploy
+=== Connect to an execution environment and deploy
 
-Before you can deploy and test your program, you need to do the following:
+Before you can deploy and test your dapp, you need to do the following:
 
-* Connect to the {IC} network either running locally in your development environment or running remotely on a subnet that you can access.
+* Connect to either the local canister execution environment, or to the {IC} blockchain mainnet, to a subnet that you can access.
```

Comment:
I think the user does not need to bother about subnets and would thus not mention them here.